src/app/map/components/MapContainer.js

```diff
@@ -79,29 +97,124 @@ class MapContainer extends Component {
     this.state = {
       // Store containers list from backend. Each container has "id", "lat" and "lng"
       containers: [],
+      selectedId: 0,
+      selectedLon: 0,
+      selectedLat: 0,
+      infoContainer: '',
+      load: true,
     };
     this.getData = getData.bind(this);
+    this.showInfo = this.showInfo.bind(this);
   }
 
   componentDidMount() {
     this.getData();
+    this.showInfo('0');
+  }
+
+  showInfo(id, lon, lat) {
+    // axios.get(process.env.REACT_APP_CORS + process.env.REACT_APP_API_QUESTIONS)
+    const path = 'http://localhost:3000/info'.concat(id).concat('.json');
```

Comment:
This path prevents from fetching the data from another device. Please remove the http://localhost:3000 part of the path. Also, add
"proxy": "https://pis-moove-it.github.io/reciclando-web"
in package.json file (you can check that it works by setting your ip as the proxy value and accessing the page from your phone in your local network).